angular-apollo-tailwind/src/app/pull-requests/pull-requests.module.ts

```diff
@@ -26,6 +26,5 @@ import { IconsModule, PipesModule } from '@shared';
     PaginationModule,
     PipesModule,
   ],
-  exports: [PullRequestsComponent],
```

Comment:
question: Do we need to be messing with the other repo here? What's the reasoning behind removing this in this PR?

Reply:
Honestly, can't remember why I did this. will revert